src/serverenvironment.cpp

```diff
@@ -491,7 +491,7 @@ bool ServerEnvironment::removePlayerFromDatabase(const std::string &name)
 	return m_player_database->removePlayer(name);
 }
 
-bool ServerEnvironment::line_of_sight(v3f pos1, v3f pos2, float stepsize, v3s16 *p)
+bool ServerEnvironment::line_of_sight(v3f pos1, v3f pos2, float stepsize, bool nobuild, v3s16 *p)
```

Comment:
You have to either change this everywhere this is called or make this the last argument and give it a default value. Not sure if `nobuild` is a good name for it.